.changeset/quiet-chairs-lick.md

```diff
@@ -0,0 +1,8 @@
+---
+'@ag.ds-next/react': minor
```

Comment:
I do wonder if this will break the accessibility of some sites, relying on the `h3` default.

Reply:
They should be using the PageAlert component as described and have under the H1 requiring it to be an H2. I've just looked at 1 repo and they are doing that, so hopefully this just improves the accessibility of every app so far

Reply:
Nice, that makes good sense.